deeplake/integrations/mmdet/mmdet_.py

```diff
@@ -197,7 +197,7 @@ def __init__(
             tensors_dict.get("masks_tensor", None), shape=self.images[0].shape
         )
         self.bboxes = self._get_bboxes(tensors_dict["boxes_tensor"])
-        bbox_format = get_bbox_format(self.bboxes, bbox_info)
+        bbox_format = get_bbox_format(self.bboxes[0], bbox_info)
```

Comment:
@farizrahman4u @adolkhan Make sure this is tolerant to the first box being empty. That often happens.